### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by ohitstom** [MrKinau/FishingBot#106] [Q&A] [answered]
Is it possible to fish for only enchant books, and for it to chuck out anything that isnt one? Would love to farm enchants afk

**Comment by MrKinau** [comment, marked as answer]
I'm sorry. Currently there is no other way than sorting the loot using hoppers, but it's a highly requested feature and it will get implemented in the next pre-release (2.7.14). There will be a system to tell the bot to drop different item lists into different directions. Currently you can only be notified and mentioned in Discord (using the webhook feature) on specific loot.
